Targeted keywords: HTML/CSS/JS (#763)
Codecademy/docs: content/javascript/concepts/arrays/terms/indexOf/indexOf.md

## Examples

> In the example below, the index with a value of 12 is logged to the console:

Reviewer: ```suggestion
In the example below, the index for the number 12 is logged to the console:
```

Reviewer: Not sure my suggestion is ideal but the original is confusing to my ear. It suggests to me that the index = 12 as opposed to the index of the value 12 in the list
